os-eap7-launch/added/launch/filters.sh

```diff
@@ -0,0 +1,85 @@
+source $JBOSS_HOME/bin/launch/launch-common.sh
+source $JBOSS_HOME/bin/launch/logging.sh
+
+function prepareEnv() {
```

Comment:
the function keyword isn't needed, and isn't POSIX, so we shouldn't use it in new code.